src/components/Jobs/AllJobs/hook.ts

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable camelcase */
+import { useQuery } from '@tanstack/react-query';
+import config from 'config';
+import { handleApiErrors } from 'utils/handleApiErrors';
+import { handleError } from 'utils/handleUnAuthorization';
+
+const useFetchJobs = (organisationId: string | null) => {
+  const { data } = useQuery(
+    ['all_jobs'],
+    async () => {
+      let url;
+      if (organisationId) {
+        url = `${config.ApiBaseUrl}/job/all?organisation=${organisationId}`;
+      } else {
+        url = `${config.ApiBaseUrl}/job/all`;
```

Comment:
Did this change in backend, to get jobs based on organisation filter from **all** endpoint.
Because, using /job/organisation/<id> endpoint gives both - active and inactive jobs.
But on public page, we only want active jobs.